config/singlerouter_ioq_blast_simplemem.json

```diff
@@ -11,7 +11,7 @@
     }
   },
   "network": {
-    "topology": "uno",
+    "topology": "singlerouter",
```

Comment:
should this be single_router? To be consistent with fat_tree

Reply:
Good catch, fixed.